python/lsst/ip/isr/ampOffset.py

```diff
@@ -107,6 +107,11 @@ def setDefaults(self):
         doc="Source detection to add temporary detection footprints prior to amp offset calculation.",
```

Comment:
Although not strictly required, `isort` is telling me that line 31 in this file should be:

```
from lsst.pipe.base import Struct, Task
```

instead of:

```
from lsst.pipe.base import Task, Struct
```

Might be worth considering changing it up, for consistency?